app/jobs/import_csv_to_big_query_job.rb

```diff
@@ -0,0 +1,10 @@
+require 'import_csv_to_big_query'
+
```

Comment:
I think we need to add a `require 'export_tables_to_cloud_storage'` here.

Reply:
`require 'export_tables_to_cloud_storage'` is performed in `import_csv_to_big_query`.